.travis.yml

```diff
@@ -10,6 +10,7 @@ git:
 
 before_install:
   - sudo apt-get -y install codespell
+  - DICT=$(python3 -c "import os.path as op; import codespell_lib; print(op.join(op.dirname(codespell_lib.__file__), 'data', 'dictionary.txt'))") && sudo mkdir -p /usr/share/codespell && sudo ln -s $DICT /usr/share/codespell/dictionary.txt
```

Comment:
I could as well simplify this to use a hard-coded path:
```
- sudo mkdir -p /usr/share/codespell && sudo ln -s /usr/lib/python3/dist-packages/codespell_lib/data/dictionary.txt /usr/share/codespell/dictionary.txt
```
There is no strong guarantee that the Python code will always return the expected path if the structure of the `codespell` package changes in future upgrades so that doesn't make it a better solution than the hardcoded path I think. Just more complicated :-/

Comments welcome.

Reply:
A less complicated fix sounds good to me.